docs/manual.rst

```diff
@@ -7,7 +7,7 @@
 .. has_math: true
 .. author: The Nikola Team
 
-:Version: 8.0.1
+:Version: 8.0.2
```

Comment:
This is Nikola’s version (not the manual’s), and this field is updated automatically on release..